webextensions/api/scripting.json

```diff
@@ -121,6 +121,26 @@
               "safari_ios": "mirror"
             }
           },
+          "match_origin_as_fallback": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "119"
```

Comment:
I confirmed that 119 is correct:
- https://issues.chromium.org/issues/40847771
- https://chromium.googlesource.com/chromium/src/+/947d9b14b15f535f5428aa3744a295e2acdf12f8
- https://chromium.googlesource.com/chromium/src/+/947d9b14b15f535f5428aa3744a295e2acdf12f8/chrome/VERSION

Reply:
@Rob--W good to know that the reference I found in their doc was correct :-)
![image](https://github.com/mdn/browser-compat-data/assets/7352080/acf2b7e0-8ccd-460d-ac68-7ea6bb4eb336)